packages/synapse-react-client/src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -27,41 +30,33 @@
       SRC.$bootstrap-4-danger-text-color
     );
   }
+
+  // PORTALS-2597, SWC-6413: all buttons should be rounded
+  // style button directly so MuiButtons within bootstrap-4-backport class divs are styled, e.g. DiscussionThread
+  button {
+    border-radius: SRC.$button-border-radius;
+    text-transform: SRC.$button-text-transform;
+  }
+
   // For all buttons, make the box shadow more intense on hover
   .btn {
     @include hover() {
       box-shadow: SRC.$box-shadow-med;
     }
   }
 
-  // Add the 'pill' class for a pill shape
-  .btn.pill {
-    border-radius: 24px;
-  }
-
-  .btn.pill.btn-lg {
+  // Large buttons
+  .btn.btn-lg {
     font-size: 16px;
     padding: 10px 24px;
   }
 
-  // Extra-long pill shape
-  .btn.pill-xl {
-    border-radius: 24px;
+  // Extra-wide button
+  .btn.btn-wide {
     min-width: 168px;
     padding: 10px;
   }
 
-  .btn.pill-xl.btn-sm {
-    min-width: 150px;
-    min-height: 20px;
-    padding: 5px;
-  }
```

Comment:
Only applied to one button, so directly styled that button instead. See this [comment](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/126/files#r1166127561)